src/components/ProjectDetail/TaskListBoard/index.tsx

```diff
@@ -20,27 +20,20 @@ export const categories = [
 
 function TaskListBoard({ tasks }: TaskListBoardProp) {
    const [tasksMap, setTasksMap] = useState<TasksMap>({});
+
+   // add sequence of tasks
+   function addSequence(task: Task[]) {
+      task.forEach((task, index) => (task.sequence = index));
+   }
+
    useEffect(() => {
       let lists: TasksMap = {};
+      Object.keys(categories).forEach((column) => {
+         lists[column] = tasks.filter(
+            (task) => task.status === (Number(column) as TaskStatus)
+         );
+      });
```

Comment:
Tại sao e lại lấy key? lấy giá trị của categories chứ, lấy keey thì luôn là 0 đầu tiên (E có thể test lại nhé - a phát hiện ra vì k thấy task nào của cột Cancelled)

```suggestion
      categories.forEach((column) => {
         lists[column] = tasks.filter(
            (task) => task.status === column)
         );
      });
```